src/context/EarnManagerProvider/providers/yearn/components/YearnManager/YearnDeposit.tsx

```diff
@@ -568,12 +611,38 @@ export const YearnDeposit = ({ api }: YearnDepositProps) => {
               </Row>
               <Row>
                 <Row.Label>
-                  <Text translation='modals.confirm.estimatedGas' />
+                  <Text
+                    translation={
+                      state.deposit.txStatus === 'pending'
+                        ? 'modals.status.estimatedGas'
+                        : 'modals.status.gasUsed'
+                    }
+                  />
                 </Row.Label>
                 <Row.Value>
                   <Box textAlign='right'>
-                    <Amount.Fiat fontWeight='bold' value='30.00' />
-                    <Amount.Crypto color='gray.500' value='0.024' symbol='ETH' />
+                    <Amount.Fiat
```

Comment:
this component is getting a lil chonky, can we break it up in the next PR?

Reply:
Yeah, we can do that. I agree, it's getting pretty big.